swagger.yaml

```diff
@@ -476,98 +703,96 @@ paths:
-                    index:
-                      type: integer
-                      description: On-chain index of validator (if missing, validator in deposited state)
-                    public_signing_key:
-                      type: string
-                    withdrawal_credentials:
-                      type: string
-                    rewards:
-                      $ref: '#/components/schemas/RewardsAndEffectivenessResponse'
-                    fee_recipient:
-                      type: string
-                  required:
-                    - balance
-                    - rewards
-                    - withdrawal_credentials
-                    - public_signing_key
-                    - fee_recipient
+                title: StatsPartialResponse
+                type: "object"
+                properties:
+                  rewards:
+                    $ref: "#/components/schemas/RewardsAndEffectivenessResponse"
+                  nodes:
+                    $ref: "#/components/schemas/NodeAggregateResponse"
+                required:
+                  - rewards
+                  - nodes
         200:
           description: "Reward and performance data have been returned, to this day"
           content:
             "application/json":
               schema:
-                title: ValidatorsResponse
-                type: "array"
-                items:
-                  type: object
-                  properties:
-                    balance:
-                      type: string
-                      pattern: "^\\d+$"
-                      description: Validator balance (in gwei)
-                    index:
-                      type: integer
-                      description: On-chain index of validator (if missing, validator in deposited state)
-                    public_signing_key:
-                      type: string
-                    withdrawal_credentials:
-                      type: string
-                    fee_recipient:
-                      type: string
-                    rewards:
-                      $ref: '#/components/schemas/RewardsAndEffectivenessResponse'
-                  required:
-                    - balance
-                    - rewards
-                    - withdrawal_credentials
-                    - public_signing_key
-                    - fee_recipient
\ No newline at end of file
+                title: StatsFullResponse
+                type: "object"
+                properties:
+                  rewards:
+                    $ref: "#/components/schemas/RewardsAndEffectivenessResponse"
+                  nodes:
+                    $ref: "#/components/schemas/NodeAggregateResponse"
+                required:
+                  - rewards
+                  - nodes
+
+  /api/v1/validators:
+    get:
+      description: "Get rewards and aggregated node statistics for validators hosted by Chorus/Opus"
+      tags:
+        - rewards & performance
+      requestBody:
```

Comment:
not sure, i know there's a PR from mk for the validators endpoint, so it will be changed- i could remove it for now. i think customer wants to see the spec mostly for stake & unstake endpoint

Reply:
ok it's not will remove it from this spec for now & make sure it's supported in the next PR, thanks for flagging @maksimryndin